app/restore_to_file.go

```diff
@@ -145,64 +146,64 @@ func CopyFile(backingFilepath, outputFile string) error {
 }
 
 func CleanupTempFiles(outputFile string, files ...string) error {
+	defer logrus.Debugf("Cleanup Temp Files %v", files)
 	outputFilePath, err := filepath.Abs(outputFile)
 	if err != nil {
-		logrus.Errorf("find absolute path for output file=%s failed: %v", outputFile, err)
-		return err
+		return errors.Wrapf(err, "Failed find absolute path for output file=%s", outputFile)
 	}
 	for _, file := range files {
 		filePath, err := filepath.Abs(file)
 		if err != nil {
-			logrus.Errorf("find absolute path for tmp file=%s failed: %v", file, err)
-			return err
+			return errors.Wrapf(err, "Failed find absolute path for tmp file=%s", file)
 		}
 		if filePath == outputFilePath {
 			continue
 		}
 		if err := os.Remove(filePath); err != nil {
 			return err
 		}
+		if _, err := os.Stat(filePath); !os.IsNotExist(err) {
```

Comment:
Why adding this? After the previous line, the file should be gone.

Reply:
To make sure that file has been removed

Reply:
Is there any case you saw the file wasn't removed and the line above didn't error out?